features/9-ingredients.feature

```diff
@@ -0,0 +1,11 @@
+@ingredients
+Feature: Ingredients
+
+    Scenario: Create duplicate ingredient
+        Given ingredients
+            | title      |
+            | meeorder01 |
+        When create an ingredient
+            | title      |
+            | meeorder01 |
+        Then should return status code 400
```

Comment:
It should be `Then should return status code 409` because it's failed in cucumber test.

Reply:
edited